ref/general/what-is-open-liberty.adoc

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description: Open Liberty is a lightweight Java runtime for building cloud-native applications and microservices.
+:page-layout: general-reference
+:seo-title: What is Open Liberty
+:seo-description: Open Liberty is a lightweight Java runtime for building cloud-native applications and microservices.
+:page-layout: general-reference
+:page-type: general
+= What is Open Liberty
+
+Open Liberty is a lightweight Java runtime for building cloud-native applications and microservices.
+
+With Open Liberty, it's easy to add and remove modular features from the latest versions of Jakarta EE and Eclipse MicroProfile. This modular structure simplifies microservice development, so you run Just enough Application Server to support the features that your application needs. Furthermore, with Open Liberty zero migration architecture, you can upgrade to the latest version with minimal impact to your current applications and configurations. Open Liberty is compatible with the Jakarta EE 8 Full Platform and Web Profile specifications and with MicroProfile 3.0. Open Liberty is deployed on every major developer platform, including Docker, Kubernetes, and Cloud Foundry. For the latest updates about Open Liberty features and capabilities, see the link:https://openliberty.io/blog/[Open Liberty blog] and https://openliberty.io/docs/[Open Liberty docs].
```

Comment:
Change,
"With Open Liberty, it's easy to add and remove modular features from the latest versions of Jakarta EE and Eclipse MicroProfile."
to,
"With Open Liberty, you can add and remove modular features from the latest versions of Jakarta EE and Eclipse MicroProfile."
Acrolinx is flagging the word 'easy'. Also, the word 'latest' which appears thrice, If there is any way to replace it, though the third occurence "For the latest updates" should be okay I guess.

Change,
"so you run Just enough Application Server to support the features that your application needs."
to,

"so you run only the required Application Server to support the features that your application needs."

Acrolinx is flagging 'just enough'

Also,
If possible replace the word 'platform' in Open Liberty is deployed on every major developer platform. Acrolinx flags it.

Reply:
Thanks Manasi- I removed "easy" but have further comments for the others-

"Just enough Application Server" is a microservice buzz-term Alasdair requested as part of this description. Leaving it in for now

I think "latest" is ok if not referencing a specific feature or product. It's used in a general sense here.

"Platform" acrolinx recommends using this to mean "operating system". The use here is different. Not though if there is a better word for Docker, etc- which identify as Platform-as-a-service (PlaaS)

Added some more info based on sources in the issue